guides/release/tutorial/8.md

```diff
@@ -0,0 +1,159 @@
+
+```shell
+ember generate acceptance-test super-rentals
+```
+
+This is called a "generator" command in Ember CLI. Generator commands automatically create files for us based on Ember's conventions and populate them with the appropriate boilerplate content, similar to how `ember new` initially created a skeleton app for us. It typically follows the pattern `ember generate <type> <name>`, where `<type>` is the kind of thing we are generating, and `<name>` is what we want to call it.
+
+In this case, we generated an *[acceptance test][TODO: link to acceptance test]* located at `tests/acceptance/super-rentals.js`.
+
+Generators aren't required; we _could_ have created the file ourselves which would have accomplished the exact same thing. But, generators certainly save us a lot of typing. Go ahead and take a peek at the acceptance test file and see for yourself.
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        Want to save even more typing? `ember generate ...` can be shortened into `ember g ...`. That's 7 fewer characters!
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="Ember Mascot">
+  </div>
+</div>
+
+Acceptance tests are one of the few types of automated testing at our disposal in Ember. We will learn about the other types later, but what makes acceptance tests unique is that they test our app from the user's perspective &mdash; they are an automated version of the "click around and see if it works" testing we did earlier, which is exactly what we need.
+
+Let's open the generated test file and replace the boilerplate test with our own:
+
+<-- TODO: format diff -->
+```
+import { module, test } from 'qunit';
+import { click, visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+ module('Acceptance | super rentals', function(hooks) {
+  setupApplicationTest(hooks);
+
+   test('visiting /', async function(assert) {
+    await visit('/');
+
+    assert.equal(currentURL(), '/');
+    assert.dom('h2').containsText('Welcome to Super Rentals!');
+
+    assert.dom('.jumbo a.button').containsText('About Us');
+    await click('.jumbo a.button');
+
+    assert.equal(currentURL(), '/about');
+  });
+});
+```
+
+First, we instruct the test robot to navigate to the `/` URL of our app by using the `visit` _test helper_ provided by Ember. This is akin to us typing `http://localhost:4200/` in the browser's address bar and hitting the `enter` key.
+
+Because the page is going to take some time to load, this is known as an *[async](TODO: link to async)* (short for _asynchronous_) step, so we will need to tell the test robot to wait by using the `await` keyword in JavaScript. That way, it will know to wait until the page finishes loading completely before moving on to the next step.
```

Comment:
```suggestion
Because the page is going to take some time to load, this is known as an *[async](TODO: link to async)* (short for _asynchronous_) step, so we will need to tell the test robot to wait by using JavaScript's `await` keyword. That way, it will wait until the page completely finishes loading before moving on to the next step.
```